### ISSUE TITLE
NVDA does read checkbox labels when using arrow key navigation

### ISSUE THREAD

**Issue by schne324** [dequelabs/cauldron#585]
see related #584 

![b1239dfe-8813-11ec-b48d-4fdcbe1f2b4f](https://user-images.githubusercontent.com/3180185/159583673-7794af27-83cd-48fa-b40e-f66279bc270c.png)

---
<dl><dt>Description</dt><dd>Informative (static) content is not readable by a screen reader.
Screen reader does not read label, role, state of checkbox/s in down arrow key navigation. this issue occurs only when browse mode is on and user is navigating using down/up arrow key.</dd><dt>Impact</dt><dd>Critical</dd><dt>Recommendation</dt><dd></dd><dt>Source code</dt><dd>

```html
<div aria-hidden="true" class="Icon Icon--checkbox-unchecked Checkbox__overlay"><svg xmlns="http://www.w3.org/2000/svg" overflow="visible" preserveAspectRatio="none" viewBox="1 1 22 22" height="24" width="24"><path d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z" vector-effect="non-scaling-stroke" fill="currentColor"></path></svg></div>
```
</dd><dt>Checkpoint</dt><dd>Reading Order (1.3.2.a)</dd><dt>URL</dt><dd>https://cauldron.dequelabs.com/components/Checkbox</dd></dl>

**Comment by np-at** [comment]
After a little testing, I think the presence of `appearance: none` in the css rule for `.Checkbox input[type='checkbox']` is causing the node to be ignored by NVDA.  In testing, removal of `appearance: none` seems to fix the problem (tested with NVDA 2022.3.5 against Firefox 98.0.1 and chrome  99.0.4844)
